google-cloud-bigtable/lib/google/cloud/bigtable/column_family_map.rb

```diff
@@ -21,11 +21,15 @@ module Google
   module Cloud
     module Bigtable
       ##
-      # An accumulator for modifications to a table's column families. See
-      # {Table#column_families}. Modifications will be atomically applied to
-      # the table's column families. Entries are applied in order, meaning
-      # that earlier modifications can be masked by later ones (in the case
-      # of repeated updates to the same family, for example).
+      # Represents a table's column families.
+      #
+      # See {Project#create_table}, {Instance#create_table} and
+      # {Table#column_families}.
+      #
+      # Modifications will be atomically applied to the table's column families.
+      # Entries are applied in order, meaning that earlier modifications can be
```

Comment:
I don't know if this part about being applied in order is meaningful for this class. The modifications will take care of this for the user by calculating only one modification for a named column family.

Reply:
This makes more sense describing the API, but not how ColumnFamilyMap is being used to call the API.